eng/xplat-job.yml

```diff
@@ -47,24 +50,42 @@ jobs:
       agentOs: Windows_NT
 
     variables:
-      buildConfig: ${{ parameters.buildConfig }}
-      ${{ if eq(parameters.buildConfig, 'checked') }}:
-        buildConfigUpper: 'Checked'
-      ${{ if eq(parameters.buildConfig, 'debug') }}:
-        buildConfigUpper: 'Debug'
-      ${{ if eq(parameters.buildConfig, 'release') }}:
-        buildConfigUpper: 'Release'
-      archType: ${{ parameters.archType }}
-      osGroup: ${{ parameters.osGroup }}
-      osGroupName: ${{ parameters.osGroupName }}
+      - name: buildConfig
+        value: ${{ parameters.buildConfig }}
+        
+      - ${{ if eq(parameters.buildConfig, 'checked') }}:
+          - name: buildConfigUpper
+            value: 'Checked'
+
+      - ${{ if eq(parameters.buildConfig, 'debug') }}:
+          - name: buildConfigUpper
+            value: 'Debug'
+
+      - ${{ if eq(parameters.buildConfig, 'release') }}:
+          - name: buildConfigUpper
+            value: 'Release'
+
+      - name: archType
+        value: ${{ parameters.archType }}
+
+      - name: osGroup
+        value: ${{ parameters.osGroup }}
+
+      - name: osGroupName
+        value: ${{ parameters.osGroupName }}
       
       # Crossbuild specific variables.
-      ${{ if eq(parameters.archType, 'arm') }}:
-        rootfsDir: /crossrootfs/arm
-        cacRootfsDir: /crossrootfs/x86
-      ${{ if eq(parameters.archType, 'arm64') }}:
-        rootfsDir: /crossrootfs/arm64
+      - ${{ if eq(parameters.archType, 'arm') }}:
+          - name: rootfsDir
+            value: /crossrootfs/arm
+
+          - name: cacRootfsDir
```

Comment:
You can remove this - don't need the CAC_ROOFS_DIR as of https://github.com/dotnet/coreclr/pull/21034

Reply:
Will address this in a separate pr.